databases/dbvis/Portfile

```diff
@@ -41,4 +41,4 @@ destroot {
 
 livecheck.type              regex
 livecheck.url               ${homepage}/CheckForUpdate/?dbvisVersion=11.0.0&dbvisEdition=Free&channels=feature&channels=maintenance
-livecheck.regex             "www.dbvis.com/download/(\[0-9.\]+)"
+livecheck.regex             "dbvis.com/download/(\[0-9.\]+)"
```

Comment:
Periods that are not in character classes (square brackets) need to be escaped. Since you don't require any Tcl variable interpolation here, you can save yourself some backslashes by using a brace-quoted string instead of a quotation-mark-quoted string:

```
livecheck.regex             {dbvis\.com/download/([0-9.]+)}
```

Reply:
I'll chose the beauty over the risk someone would actually put something else than a dot in between "dbvis" and "com" (my regex is actually pretty fluent), and therefore very much appreciate the brace-delimiting option you suggest to avoid even more backslashes.